File: Repetitions.py
def count_repetition(s):
    n = len(s)
    max_count = 0
    l = 0

    while l < n:
        r = l
        while r < n and s[l] == s[r]:
            r += 1
        max_count = max(max_count, r - l)
        l = r

    return max_count

def main():
    s = input()
    print(count_repetition(s))

main()

File: Repetitions.java
import java.util.*;

public class Repetitions {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        String str = sc.nextLine();
        System.out.println(countRepetition(str));
        sc.close();
    }

    private static int countRepetition(String str) {
        int n = str.length();
        int maxCount = 0;
        int r;

        for (int l = 0; l < n; l = r) {
            r = l;

            while (r < n && str.charAt(l) == str.charAt(r)) {
                r++;
            }
            maxCount = Math.max(maxCount, r - l);
        }

        return maxCount;
    }
}
